Headlines/src/Managers/ImageCacheManager.swift

```diff
@@ -8,36 +8,40 @@
 
 import Foundation
 import UIKit
-import SDWebImage
+import Kingfisher
 
 class ImageCacheManager {
 
     static let shared = ImageCacheManager()
+    private let cache = ImageCache(name: "canillitapp.cache")
+    private var cachedImages = [String: UIImage]()
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/a5cc79d8b0bc8bc433e47d6a58e00624ade77840/Headlines/src/Managers/ImageCacheManager.swift#L18"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>ImageCacheManager.swift:18</code></td>
    </tr>
  </tbody>
</table>